config/crd-hub/bases/hub.kmm.sigs.x-k8s.io_managedclustermodules.yaml

```diff
@@ -2362,6 +2362,18 @@ spec:
                                 description: ModuleName is the name of the Module
                                   to be loaded.
                                 type: string
+                              modulesLoadingOrder:
+                                description: 'ModulesLoadingOrder defines the dependency
```

Comment:
Please update the CRD description to match the comment.

Reply:
fixed